tests/BoardTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace GameOfLife\tests;
+
+use PHPUnit\Framework\TestCase;
+use GameOfLife\Board;
+use GameOfLife\Field;
+
+
+class BoardTest extends TestCase
+{
+    public function testConstructedBoardIsEmpty()
+    {
+        $_board = new Board(50, 25);
+        $isEmpty = true;
+
+        for ($y = 0; $y < $_board->height(); $y++)
+        {
+            for ($x = 0; $x < $_board->width(); $x++)
+            {
+                if ($_board->boardValue($x, $y) != 0)
+                {
+                    $isEmpty = false;
+                }
+            }
+        }
+
+        $this->assertTrue($isEmpty);
+
+        return $_board;
+    }
+
+    /**
+     * @depends clone testConstructedBoardIsEmpty
+     * @param Board $_board
+     */
+    public function testBoardElements(Board $_board)
+    {
+        $this->assertEquals(50, $_board->width());
+
+        $this->assertEquals(25, $_board->height());
+    }
+
+    /**
+     *
+     */
+    public function testCreateFromArrayWorks()
+    {
+        $board = new Board(5, 5);
+        $board->setBoardValue(1, 1, 1);
+        $board->setBoardValue(3, 1, 1);
+
+        $board->setBoardValue(0, 3, 1);
+        $board->setBoardValue(1, 4, 1);
+        $board->setBoardValue(2, 4, 1);
+        $board->setBoardValue(3, 4, 1);
+        $board->setBoardValue(4, 3, 1);
+
+        $testBoard = Board::createFromArray
+        ([
+            [0, 0, 0, 0, 0],
+            [0, 1, 0, 1, 0],
+            [0, 0, 0, 0, 0],
+            [1, 0, 0, 0, 1],
+            [0, 1, 1, 1, 0]]
+        );
+
+        $this->assertTrue($board->isEqualTo($testBoard));
+    }
+
+    public function testCreateFromArrayInSmallSize()
+    {
+        $board = new Board(3, 3);
+        $board->setBoardValue(0, 2, 1);
+
+        $testBoard = Board::createFromArray
+        ([
+                [0, 0, 0],
+                [0, 0, 0],
+                [1, 0, 0]
+        ]);
+
+        $this->assertTrue($board->isEqualTo($testBoard));
```

Comment:
Using a method of the tested class in an assertion is bit questionable.
Imagine the isEqualTo() has a bug and returns wrong values. Then this test will probably fail and the developer will think the createFromArray() is broke, but in fact it is not, it is isEqualTo() that is broken.
So better would be to make the assertion manually in this class. At least you should move the isEqualTo() tests above this method so that they fail FIRST if there is an error in this method.